src/components/LayoutTree.tsx

```diff
@@ -434,14 +441,20 @@ const TreeNode = ({
   // Increasing margin-right of thought for filling gaps and moving the thought to the left by adding negative margin from right.
   const marginRight = isTableCol1 ? xCol2 - (width || 0) - x - (bulletWidth || 0) : 0
 
+  // Speed up the tree-node's transition by 50% on New (Sub)Thought only.
+  const isNewThought = value === ''
```

Comment:
@raineorshine - Can you please clarify what is intended in this part of #1790? I believe the current functionality is incorrect, but it isn't obvious to me what the desired animations/durations are.

> Speed up the tree-node's transition (normally layoutNodeAnimationDuration) by 50% on New (Sub)Thought only.

This logic currently affects the animation duration when indenting to convert an empty thought into a subthought. It does not affect the animation when indenting a non-empty thought or when using the `newSubthought` action. Here is a demo of the three, with `layoutNodeAnimationDuration` increased for a more noticeable difference:

https://github.com/user-attachments/assets/e052d3b8-df04-4b64-b9e6-d10e7d358584